images/redis/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM redis:alpine
+
+ENV S6_VERSION 2.0.0.1
+ENV EXPORTER_VERSION 1.24.0
+ENV S6_BEHAVIOUR_IF_STAGE2_FAILS 2
+
+RUN apk update && apk add wget
+
+RUN wget https://github.com/just-containers/s6-overlay/releases/download/v${S6_VERSION}/s6-overlay-amd64.tar.gz -O /tmp/s6-overlay-amd64.tar.gz \
```

Comment:
`curl` would do the same and it may be installed already, avoiding the need to install `wget`. (`curl -L -o /tmp/s6-overlay-amd64.tar.gz https://...`). But as discussed those are images for debugging or community use so not super important to fine-tune them (I don't know about the Artesca situation, it's possible they could eventually be used in the product from an 8.x branch, following the component decoupling philosophy).

Reply:
its the alpine version of the redis; curl would need to be installed anyway

Reply:
wget should already be present as it's provided by busybox